providers/base/tests/test_bt_list_adapters.py

```diff
@@ -0,0 +1,32 @@
+import unittest
+from unittest.mock import patch, mock_open
+from pathlib import Path
+
+import bt_list_adapters
+
+
+class BTTests(unittest.TestCase):
```

Comment:
Missing `main` and `unhappy_main` tests, I would add them

Reply:
Tests added